sample-input/point-source/point-source.py

```diff
@@ -8,9 +8,9 @@
 
 num_threads = options.getNumThreads()
 track_spacing = options.getTrackSpacing()
-num_azim = options.getNumAzimAngles()
+num_azim = 32
```

Comment:
Do you think the parameters here should be the new defaults for this example script?